<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="classAdd1Widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>431</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QLabel" name="otherLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalDayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total days</string>
    </property>
   </widget>
   <widget class="QWidget" name="dayUseWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="dayUseLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Day use</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="dayUseSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="totalDateLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="regisdateEdit">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="classNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Class name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="classNameLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="regisDayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Registration day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otherLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>otherLabel</zorder>
   <zorder>totalDayLabel</zorder>
   <zorder>dayUseWidget</zorder>
   <zorder>totalDateLineEdit</zorder>
   <zorder>regisdateEdit</zorder>
   <zorder>classNameLabel</zorder>
   <zorder>classNameLineEdit</zorder>
   <zorder>regisDayLabel</zorder>
   <zorder>otherLineEdit</zorder>
  </widget>
  <widget class="QWidget" name="courseInfoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>411</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="courseInfoLayout"/>
   </widget>
  </widget>
  <widget class="QWidget" name="classAdd2Widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit, delete or add member in table directly </string>
    </property>
   </widget>
   <widget class="QLabel" name="memberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
   </widget>
   <widget class="QTableView" name="addMemberTable">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>label_3</zorder>
   <zorder>memberLabel</zorder>
   <zorder>addMemberTable</zorder>
  </widget>
  <widget class="QWidget" name="classAdd3Widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>519</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QLabel" name="courseClassLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="courseComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chooseCourseLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose course</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>courseClassLabel</zorder>
   <zorder>courseComboBox</zorder>
   <zorder>chooseCourseLabel</zorder>
  </widget>
  <widget class="QPushButton" name="enableClass2Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>250</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable member edit</string>
   </property>
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/myclass/Resources/TurnOn_icon.png</normaloff>:/myclass/Resources/TurnOn_icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>620</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/myclass/Resources/cancel_icon.png</normaloff>:/myclass/Resources/cancel_icon.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="courseInfoLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="enableClass1Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable class info edit</string>
   </property>
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/myclass/Resources/TurnOn_icon.png</normaloff>:/myclass/Resources/TurnOn_icon.png</iconset>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>20</width>
     <height>591</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>620</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/myclass/Resources/save_icon.png</normaloff>:/myclass/Resources/save_icon.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="enableClass3Button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable course edit</string>
   </property>
   <property name="icon">
    <iconset resource="myclass.qrc">
     <normaloff>:/myclass/Resources/TurnOn_icon.png</normaloff>:/myclass/Resources/TurnOn_icon.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
